frontend/src/components/grantaccess/ForestClientCard.vue

```diff
@@ -1,72 +1,134 @@
-                >
-                    <p>
-                        <label>Organization name: </label>&nbsp;<span
-                            class="organization-name"
-                            >{{ props.text }}</span
+                <div class="w-100">
+                    <div class="content-wrapper">
+                        <Icon
+                            class="row flex-grow-0 custom-carbon-icon-checkmark--filled"
+                            icon="checkmark--filled"
+                            :size="IconSize.small"/>
+                        <p
+                            class="col flex-grow-0 client-id-wrapper"
+                        >
+                            <label class="col">Client ID: </label>
+                            <span class="col">
+                                {{ props.forestClientData?.forest_client_number }}
+                            </span>
+                        </p>
+                        <p
+                            class="col"
+                            v-if="props.forestClientData?.client_name != 'Doesn\'t exist'"
                         >
-                    </p>
+                            <label class="row">Organization name: </label>
+                            <span class="organization-name row">
+                                {{ props.forestClientData?.client_name }}
+                            </span>
+                        </p>
+                        <p
+                            class="col org-status-wrapper"
+                        >
+                            <label class="row status">Organization status: </label>
+                            <Tag
+                                class="custom-tag"
+                                :text="props.forestClientData?.status?.description"
+                            />
+                        </p>
+                        <Button class="btn-trash">
+                            <Icon
+                                class="row custom-carbon-icon--trash-can"
+                                icon="trash-can"
+                                :size="IconSize.small"
+                                title="Remove client"
+                            />
+                        </Button>
+                    </div>
                 </div>
-                <span class="invalid" v-if="props.status.status_code !== 'A'"
-                    >Please enter an active Forest Client ID</span
-                >
             </template>
         </Card>
     </div>
 </template>
 
 <style lang="scss" scoped>
 @import '@/assets/styles/styles.scss';
-p {
-    margin: 0;
-    padding: 0 !important;
-    display: inline-flex;
-}
 
 p > label {
     margin-top: .25rem;
 }
-.invalid {
-    margin-top: 0.25rem;
-    font-size: 0.875em;
-    color: #dc3545;
+
+p {
+    display: flex;
+    flex-direction: column;
+}
+
+p * {
+    align-self: baseline !important;
+}
+
+.col {
+    align-self: center;
+}
+
+.custom-card {
```

Comment:
We have this style in the user card as well, maybe we should put it in one of the style file under assets/styles

Reply:
Nice catch! I'll fix this!